discord/template.py

```diff
@@ -147,9 +155,9 @@ async def create_guild(self, name, region=None, icon=None):
 
         Raises
         ------
-        :exc:`.HTTPException`
+        HTTPException
             Guild creation failed.
-        :exc:`.InvalidArgument`
+        InvalidArgument
```

Comment:
This can be reverted https://github.com/Rapptz/discord.py/pull/6467#issuecomment-783812227